Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
+MAINTAINER Avishkar Gupta <avgupta@redhat.com>
+
+RUN yum install -y epel-release && \
+    yum install -y zip && \
+    yum install -y python-pip python-devel gcc && \
+    yum -y install tkinter && \
+    yum-config-manager --disable testing-devtools-2-centos-7 && \
+    yum -y install gcc-c++.x86_64 && \
+    yum clean all
+
+
+# --------------------------------------------------------------------------------------------------
+# install python packages
+# --------------------------------------------------------------------------------------------------
+COPY ./hack/requirements.txt /
+RUN pip install -r /requirements.txt && rm /requirements.txt
+RUN pip install pomegranate==0.7.3
```

Comment:
Why is this here and not in `requirements.txt` ?

Reply:
There's an upstream bug with dependency resolution for this package, we tried doing it in Kronos: https://github.com/fabric8-analytics/fabric8-analytics-stack-analysis/pull/68

Reply:
OK, then put a comment there so other people don't have to ask the same question.